docs/src/main/sphinx/connector/exasol.rst

```diff
@@ -0,0 +1,320 @@
+    - ``DECIMAL(p, s)``
+    -  See :ref:`exasol number mapping`
+  * - ``DOUBLE PRECISION``
+    - ``REAL``
+    -
+  * - ``VARCHAR(n)``
+    - ``VARCHAR(n)``
+    -
+  * - ``CHAR(n)``
+    - ``CHAR(n)``
+    -
+  * - ``DATE``
+    - ``DATE``
+    -
+  * - ``TIMESTAMP``
+    - ``TIMESTAMP``
+    -
+
+No other types are supported.
+
+Trino to Exasol type mapping
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Trino supports creating tables with the following types in an Exasol database.
+The table shows the mappings from Trino to Exasol data types:
+
+.. list-table:: Trino to Exasol Type Mapping
+  :widths: 30, 25, 50
+  :header-rows: 1
+
+  * - Trino type
+    - Exasol database type
+    - Notes
+  * - ``TINYINT``
+    - ``DECIMAL(3, 0)``
+    -
+  * - ``SMALLINT``
+    - ``DECIMAL(9, 0)``
+    -
+  * - ``INTEGER``
+    - ``DECIMAL(18, 0)``
+    -
+  * - ``BIGINT``
+    - ``DECIMAL(36 ,0)``
+    -
+  * - ``DECIMAL(p, s)``
+    - ``DECIMAL(p, s)``
+    -
+  * - ``REAL``
+    - ``DOUBLE PRECISION``
+    -
+  * - ``DOUBLE``
+    - ``DOUBLE PRECISION``
+    -
+  * - ``VARCHAR``
+    - ``VARCHAR(2000000)``
+    -
+  * - ``VARCHAR(n)``
+    - ``VARCHAR(n)``
+    - See :ref:`exasol character mapping`
+  * - ``CHAR(n)``
+    - ``CHAR(n)``
+    - See :ref:`exasol character mapping`
+  * - ``DATE``
+    - ``DATE``
+    -
+  * - ``TIMESTAMP``
+    - ``TIMESTAMP``
+    -
+
+No other types are supported.
+
+.. _exasol number mapping:
+
+Mapping numeric types
+^^^^^^^^^^^^^^^^^^^^^
+
+An Exasol ``DECIMAL(p, s)`` maps to Trino's ``DECIMAL(p, s)`` and vice versa
+except in these conditions:
+
+- No precision is specified for the column (example: ``DECIMAL`` or
+  ``DECIMAL(*)``).
+- Scale (``s`` ) is greater than precision.
+- Precision (``p`` ) is greater than 36.
+- Scale is negative.
+
+.. _exasol character mapping:
+
+Mapping character types
+^^^^^^^^^^^^^^^^^^^^^^^
+
+Trino's ``VARCHAR(n)`` maps to ``VARCHAR(n)`` and vice versa if ``n`` is no greater
+than 2000000. Exasol does not support longer values.
+If no length is specified, the connector uses 2000000.
+
+Trino's ``CHAR(n)`` maps to ``CHAR(n)`` and vice versa if ``n`` is no greater than 2000.
+Exasol does not support longer values.
+
+.. include:: jdbc-type-mapping.fragment
+
+.. _exasol-sql-support:
+
+SQL support
+-----------
+
+The connector provides read access and write access to data and metadata in
+Exasol. In addition to the :ref:`globally available <sql-globally-available>`
+and :ref:`read operation <sql-read-operations>` statements, the connector
+supports the following statements:
+
+* :doc:`/sql/delete`
+* :doc:`/sql/truncate`
+* :doc:`/sql/create-table`
+* :doc:`/sql/drop-table`
+* :doc:`/sql/alter-table`
+* :doc:`/sql/comment`
+
+SQL INSERT
+^^^^^^^^^^
+
+Please note that ``INSERT`` and ``CREATE TABLE AS`` is not supported because
+this would require `parallel connections with the Exasol JDBC driver
+<https://exasol.my.site.com/s/article/Parallel-connections-with-JDBC?language=en_US>`_
```

Comment:
Actually .. now reading this .. the whole SQL Insert section should be removed .. this is irrelevant to users.

However ultimately we should figure out how to work around this..